apps/client/src/pages/room/Room.page.tsx

```diff
@@ -0,0 +1,5 @@
+import { ReactElement } from 'react';
+
+export function Room(): ReactElement {
```

Comment:
Can you do a small background research on the return type of react components?
Need you to argue about whether we should use
1. ReactElement
2. JSX.ELement
3. ReactNode
4. [something else?]
as the return type of react components

Reply:
Discussed to use type ReactElement on components (atoms, molecules, organisms) and type ReactNode on templates (Race track, Leaderboard...)